bahmni-lite/docker-compose.yml

```diff
@@ -68,6 +68,8 @@ services:
       MAIL_USER: ${MAIL_USER}
       MAIL_PASSWORD: ${MAIL_PASSWORD}
       OMRS_DOCKER_ENV: ${OPENMRS_DOCKER_ENV}
+      OPENMRS_HOST: ${OPENMRS_HOST:?}
```

Comment:
Why are these variables needed? host and port of the openmrs within openmrs? Are we making any request from within the application?

Reply:
Can be removed not required anymore because we removed the internal application api call.